content/docs/04.5-devx.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Palette Dev Engine"
+metaTitle: "Palette Dev Engine "
+metaDescription: "Explore Palette Dev Engine"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+
+# Overview
+
+Palette provides a developer persona of the Palette operations in addition to the DevOps persona of creating, deploying, and managing applications in Palette. This persona is called Palette Dev Engine (PDE); the experience developers have while building, maintaining, testing, deploying, and monitoring applications through Palette. A developer can easily switch between these two personas - `App Mode and Cluster Mode`- through the Palette UI. We offer the PDE from two wide perspectives:
+
+<br />
+
+
+# Quick Start with Palette Dev Engine
+
+The quick-start developer experience with Palette Dev Engine is exceptionally straightforward. We provide a smooth developer experience where anyone can log in to Palette Dev Engine and deploy their first application in just **three** steps.
+
+|    |  |
+|----|---|
+|**Step 1** |[Login to Palette Dev Engine](/devx#loginoptions)|
+|**Step 2** |Select an App Profile (we do offer a set of out-of-the-box profiles) or [Create an App Profile](/devx/app-profile) based on your use case|
+|**Step 3** |Using the App Profile [deploy your application](/devx/apps)| 
+
+Refer the below links to explore the Palette Dev Engine in depth ....
+
+<br />
+
+* **Sign up to Manage Apps**: As a free tier offering from Palette for Developers to [Land and Expand](/devx/dev-land-explore).
+
+
+* **Sign up to Manage Clusters**: For [Enterprise Developers](/devx/enterprise-user) full Palette capabilities.
+
+# Login Options
+
+Palette provides two options of developer log in to the Palette Dev Engine console:
+
+<br />
+
+## Social Sign-In 
+
+The developer can sign in directly to the Palette Dev Engine console using their social sign-in credentials, such as Google or GitHub. These developers are at benefit from using our complimentary resources to either:
+
+<br />
+
+* Provision a nested cluster to get full access to all the namespaces
+
+* Deploy an application using an app profile that would spin up a nested cluster
+
+In either case, they are governed by the quota set by Palette. There is no limit to the number of application profiles they can be modelled.
+
+<br />
+
+### Enable Social Sign-In
+
+To enable social Sign In, the user as a Palette tenant admin need to set up SSO authentication in the Palette console.
```

Comment:
Since they are by default enabled, may be it can reside into some other page with heading `Disable Social Sign In`

Reply:
sure @tiwarisumit3